app/controllers/comments_controller.rb

```diff
@@ -59,13 +66,52 @@ def destroy
     end
   end
 
+  def stimulus
+    @formType = "horizontal"
```

Comment:
We might not need this instance variable @formType by using action_name instead.